bubblegum/js/tests/main.test.ts

```diff
@@ -21,8 +27,8 @@ function keypairFromSeed(seed: string) {
     return Keypair.fromSeed(expandedSeed.slice(0, 32));
 }
 
-const LOG_WRAPPER_ID = new PublicKey("noopb9bkMVfRPU8AsbpTUg8AQkHtKwMYZiFUjNRtMmV");
-const COMPRESSION_PROGRAM_ID = new PublicKey("cmtDvXumGCrqC1Age74AVPhSRVXJMd8PJS91L8KbNCK");
+// const LOG_WRAPPER_ID = new PublicKey("noopb9bkMVfRPU8AsbpTUg8AQkHtKwMYZiFUjNRtMmV");
+// const COMPRESSION_PROGRAM_ID = new PublicKey("cmtDvXumGCrqC1Age74AVPhSRVXJMd8PJS91L8KbNCK");
```

Comment:
can these be removed if not needed any more

Reply:
Removed